modules/pool/pool.gql

```diff
@@ -317,6 +317,7 @@ type GqlPoolDynamicData {
     fees24h: BigDecimal!
     yieldCapture24h: BigDecimal!
     apr: GqlPoolApr!
```

Comment:
how about replacing that just with a sum as a number? it would be the same value we use for sorting

Reply:
it's already there under poolBase